backend/molgenis-emx2-sql/src/main/resources/org/molgenis/emx2/sql/migration26.sql

```diff
@@ -0,0 +1,30 @@
+DO $$
+    DECLARE
+        rec RECORD;
+        alter_stmt TEXT;
+    BEGIN
+        FOR rec IN
+            SELECT
+                n.nspname AS schema_name,
+                p.proname AS function_name,
+                pg_get_function_identity_arguments(p.oid) AS args
+            FROM
+                pg_proc p
+                    JOIN pg_namespace n ON p.pronamespace = n.oid
+                    JOIN pg_language l ON p.prolang = l.oid
+            WHERE
+                p.prorettype = 'pg_catalog.trigger'::regtype
+              AND n.nspname NOT IN ('pg_catalog', 'information_schema', '_SYSTEM_', 'MOLGENIS')
```

Comment:
Here I exclude the system schema's. I think this should cover it. Please review carefully. This worked for me locally.